src/renderer/components/+preferences/preferences.tsx

```diff
@@ -284,6 +316,43 @@ export class Preferences extends React.Component {
             }
           </section>
         )}
+        {this.activeTab == Pages.Editor && (
+          <>
```

Comment:
Please move this contents to separate `<EditorSettings>` component to prevent Preferences to grow up.